apps/main/src/lend/lib/apiLending.ts

```diff
@@ -497,9 +498,22 @@ const user = {
             market.userPrices(),
             market.userBandsBalances(),
             market.oraclePriceBand(),
-            market.userLoss(),
           ])
 
+        // Fetch user loss separately to prevent prices-api dependency from blocking contract read data
+        let loss: UserLoss
+        try {
+          loss = await market.userLoss()
+        } catch (error) {
+          console.error('Failed to fetch user loss:', error)
+          loss = {
```

Comment:
could put this initialization above the `try` i guess

Reply:
Please let's stop using 0 for fallback :sob: 
what f the user has some loss and we report 0? It's just wrong!
Any place using these values might want to handle it differently

Reply:
On error loss remains `undefined` now, and in those cases a '?' should be shown